frontend/src/lib/MlmdUtils.ts

```diff
@@ -169,3 +172,40 @@ function getStringValue(value?: string | number | Struct | null): string | undef
   }
   return value;
 }
+
+/**
+ * @throws error when network error or invalid data
+ */
+export async function getOutputArtifactsInExecution(execution: Execution): Promise<Artifact[]> {
```

Comment:
nit: remove the `getOutputArtifactsInExecution` method in `frontend/src/lib/OutputArtifactLoader.ts` file?

Reply:
Good point! Done